pyomo/contrib/mindtpy/nlp_solve.py

```diff
@@ -51,7 +52,8 @@ def solve_NLP_subproblem(solve_data, config):
                + (0 if c.lower is None else c.lower))
         sign_adjust = 1 if value(c.upper) is None else -1
         fix_nlp.tmp_duals[c] = sign_adjust * max(0,
-                sign_adjust * (rhs - value(c.body)))
+                                                 sign_adjust*(rhs - value(c.body)))
```

Comment:
If you are going to reformat (PEP8) this line, I would recommend:
```python
fix_nlp.tmp_duals[c] = sign_adjust * max(
    0, sign_adjust * (rhs - value(c.body)))
```

Reply:
Thanks. I have changed it in the latest commit.